.github/workflows/release.yaml

```diff
@@ -44,15 +45,24 @@ jobs:
       - name: Install tool dependencies
         uses: jdx/mise-action@v2
 
-      - name: Package CRDs
+      - name: Package kubernetes manifests & CRDs
         run: |
           cat deploy/charts/prefect-operator/crds/*.yaml > prefect-crds.yaml
+          helm dependency build deploy/charts/prefect-operator
+          helm template prefect-operator deploy/charts/prefect-operator \
+          --include-crds --set operator.image.tag=${{ github.ref_name }} \
+          > prefect-operator.yaml
+          yq -i 'del(.metadata.labels."app.kubernetes.io/managed-by")' prefect-operator.yaml
+          yq -i 'del(.metadata.labels."helm.sh/chart")' prefect-operator.yaml
+          yq -i 'del(.spec.template.metadata.labels."app.kubernetes.io/managed-by")' prefect-operator.yaml
+          yq -i 'del(.spec.template.metadata.labels."helm.sh/chart")' prefect-operator.yaml
+          yq -i '.metadata.labels."app.kubernetes.io/version" = "${{ github.ref_name }}"' prefect-operator.yaml
+          yq -i '.spec.template.metadata.labels."app.kubernetes.io/version" = "${{ github.ref_name }}"' prefect-operator.yaml
```

Comment:
I ran all of these `yq` statements locally to build the manifest, and I think we'll have to be more specific about where these fields get injected. At the moment they get injected on every document, but we need to select exactly where they need to go (just for the Deployment, right)?

Otherwise I get an error on apply:

> strict decoding error: unknown field "spec.template"

For example, it's putting that metadata on RoleBindings:

<img width="407" alt="image" src="https://github.com/user-attachments/assets/5552a6a9-1e93-40f1-b493-077538935a80">

Reply:
good catch -- let me dig in